apps/portals/src/portal-components/challengeportal/ChallengeDetailPageWrapper.tsx

```diff
@@ -11,7 +15,23 @@ import { AppUtils, ChallengeDetailPage } from 'synapse-react-client'
  */
 const ChallengeDetailPageWrapper = () => {
   const projectId = AppUtils.useQuerySearchParam('id')
-  if (projectId) {
+  const { data, isLoading } = SynapseQueries.useGetEntityBundle(
+    projectId,
+    undefined,
+    {
+      includeAnnotations: true,
+    },
+    {
+      enabled: !!projectId,
+    },
+  )
+  if (isLoading) {
+    return <></>
+  }
+  if (
+    !!projectId &&
+    data?.annotations.annotations['Status'].value[0] == 'Active'
```

Comment:
Are we guaranteed to have a `'Status'` annotation? Consider optional chaining in case it's missing, to prevent an error thrown by accessing `undefined.value`
```suggestion
    data?.annotations.annotations['Status']?.value[0] == 'Active'
```

Reply:
Good call.  It's possible that they are sent a direct link to this challenge detail page (from Open Challenges, for example) instead of browsing from the home page, so this case is definitely possible.  Made the change